includes/class-freemius.php

```diff
@@ -7544,6 +7545,45 @@ function _activate_plugin_event_hook() {
             $this->_storage->was_plugin_loaded = true;
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.8.0
+         *
+         * @param string $plugin
+         * @param bool   $network_wide
+         */
+        function _activated_plugin( $plugin, $network_wide ) {
+            // We cannot rely on fs_remove_sdk_reference_by_basename() for reordering, as the latest SDK reference might be associated with a different product, leading to the reordering of that product instead of the intended one.
+            $this->move_free_plugin_to_end_of_active_plugins( $this->_free_plugin_basename );

```

Comment:
I suggest not to pass the `$this->_free_plugin_basename` as the function can already get it from the instance. (Just for code readability)